client/app/scripts/superdesk-packaging/packaging.js

```diff
@@ -467,7 +467,7 @@
             link: function(scope) {
                 scope.data = null;
                 scope.error = null;
-                scope.type = scope.item.type || scope.item.itemClass.split(':')[1];
```

Comment:
This line was causing an error, `item` did not have neither `type` nor the `itemClass` property - for the packages created from RSS feeds at least. But does it behave differently for Reuters packages?